Tasks/HelmInstallerV0/src/helmtoolinstaller.ts

```diff
@@ -41,6 +41,8 @@ async function verifyHelm() {
         var helmTool = tl.tool(helmToolPath);
         helmTool.arg("init");
         helmTool.arg("--client-only");
+        helmTool.arg("--stable-repo-url");
```

Comment:
This would work only for the latest versions right?

Reply:
Yes, looks to be specific to version specific problem.

Reply:
No, since the specified the url is valid, it doesn't matter what helm version is calling that url. It works with older versions also, I have verified it.